js-peer/src/context/ctx.tsx

```diff
@@ -20,7 +19,12 @@ interface WrapperProps {
 // This is needed to prevent libp2p from instantiating more than once
 let loaded = false
 export function AppWrapper({ children }: WrapperProps) {
-  const [libp2p, setLibp2p] = useState<Libp2pType>()
+  const [libp2p, setLibp2p] = useState<
+    Libp2p<{
```

Comment:
What do you think about adding a type alias for the libp2p type (with pubsub and identify) used by this peer and reusing both here and in `js-peer/src/lib/libp2p.ts`?

Reply:
Was having some stress keeping TS happy so will take another look at this